optifit backend/app.py

```
@@ -37,108 +60,326 @@ def error_response(message, status_code):
             "video_url": video_url
         }
         
+        logger.info(f"Started {exercise_type} processing job: {job_id}")
         return jsonify(response_data)
     
-    except APIError as e:
-        logger.error(f"API Error in {request.endpoint}: {e.message}", exc_info=True)
-        return error_response(e.message, e.status_code)
-    
-
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)